CHANGE_LOG

```diff
@@ -1,3 +1,242 @@
+Version 0.55.0 (10 December, 2021)
+----------------------------------
+
+This release includes a significant number important dependency upgrades along
+with a number of new features and bug fixes.
+
+Highlights of core dependency upgrades:
+
+* Support for Python 3.10
+* Support for NumPy 1.21
+
+Python language support enhancements:
+
+* Experimental support for ``isinstance``.
+
+NumPy features/enhancements:
+
+The following functions are now supported:
+
+* ``np.broadcast_to``
+* ``np.float_power``
+* ``np.cbrt``
+* ``np.logspace``
+* ``np.take_along_axis``
+* ``np.average``
+* ``np.argmin`` gains support for the ``axis`` kwarg.
+* ``np.ndarray.astype`` gains support for types expressed as literal strings.
+
+Highlights of core changes:
+
+* For users of the Numba extension API, Numba now has a new error handling mode
+  whereby it will treat all exceptions that do not inherit from
+  ``numba.errors.NumbaException`` as a "hard error" and immediately unwind the
+  stack. This makes it much easier to debug when writing ``@overload``\s etc
+  from the extension API as there's now no confusion between Python errors and
+  Numba errors. This feature can be enabled by setting the environment
+  variable: ``NUMBA_CAPTURED_ERRORS='new_style'``.
+* The threading layer selection priority can now be changed via the environment
+  variable ``NUMBA_THREADING_LAYER_PRIORITY``.
+
+Highlights of changes for the CUDA target:
```

Comment:
@gmarkall As code owner for CUDA, any chance you could suggest an addition for the recently approved half precision support: https://github.com/numba/numba/pull/7460 ? Many thanks.

Reply:
i added in https://github.com/numba/numba/pull/7644/commits/b6bcb18523289b79aaa39d3445ec5eea2274a8f1